manifests/interface.pp

```diff
@@ -634,9 +634,17 @@
     },
     default => $peerntp,
   }
-  $manage_ipaddr = $ipaddr ? {
-    ''      => $ipaddress,
-    default => $ipaddr,
+  case $ipaddr {
+    '': { $manage_ipaddr = $ipaddress}
+    default: {
+      if $ipaddr =~ /^([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})\/([0-9]|[1-2][0-9]|3[0-2])$/ {
+        $manage_ipaddr = $1
```

Comment:
Where are set $1 and $2?

Reply:
They are coming from the regular expression. It is trying to extract IP into $1 and prefix into $2 if possible. 
So that ipaddress can be set in debian or CIDR way: 1.1.1.1/24 -> manage_ipaddr=1.1.1.1 and manage_prefix=24. 

It will override prefix for RH, but only if IP address is in a CIDR format.